src/views/Stake/ExternalStakePool.tsx

```diff
@@ -133,41 +87,21 @@ const StakePool = ({ pool, isLoading }: { pool: ExternalPoolwBalance; isLoading:
   return (
     <Box style={{ gap: theme.spacing(1.5) }} className={styles.stakePoolsWrapper}>
       <Box sx={{ display: "flex", alignItems: "center" }}>
-        <MultiLogo icons={pool.icons} />
+        <TokenStack tokens={pool.icons} />
         <Typography gutterBottom={false} style={{ lineHeight: 1.4, marginLeft: "10px" }}>
           {pool.poolName}
         </Typography>
       </Box>
-      <Typography gutterBottom={false} style={{ lineHeight: 1.4 }}>
-        {/* {pool.apy} */}
-      </Typography>
       <Typography gutterBottom={false} style={{ lineHeight: 1.4 }}>
         {!pool.tvl ? <Skeleton width={30} /> : pool.tvl}
       </Typography>
       <Typography gutterBottom={false} style={{ lineHeight: 1.4 }}>
-        {isLoading ? <Skeleton width={30} /> : connected && pool.userBalance ? `${pool.userBalance} LP` : ""}
+        {!pool.userBalance ? <Skeleton width={30} /> : connected && pool.userBalance ? `${pool.userBalance} LP` : ""}
```

Comment:
why is this desktop balance skeleton check different from Mobile balance skeleton check? see above